src/api/wampApi.ts

```diff
@@ -0,0 +1,61 @@
+import autobahn = require('autobahn');
+
+export class WampApi {
+  session: any;
+
+  constructor(url: string, realm: string) {
+    this.connect(url, realm);
```

Comment:
not sure about this implicit behavior. For me, it's better to call `connect` explicitly